Add user roles documentation (#120)
netbirdio/docs: src/pages/how-to/add-users-to-your-network.mdx

    This is a limitation that is likely to be removed in future versions.
</Note>

## Manage user roles
NetBird has three user roles - `Owner`, `Admin`, and `User`. The roles allow you to control the level of access to the management API of your account.

> - `Owner` role - has full access to the account and can manage all aspects of the account. This role can not be assigned to multiple users.

Reviewer: Mention account deletion and add an account deleteion doc reference.  
`This role can not be assigned to multiple users.`
There can be only one account owner in NetBird.

> - `Owner` role - has full access to the account and can manage all aspects of the account. This role can not be assigned to multiple users.

Reviewer: ```suggestion
- `Owner` role - has full access to the account and can manage all aspects of the account. There can be only one account owner in NetBird. Users with the owner role can delete their organization account. See the [Delete NetBird account](/how-to/delete-account) section.
```

> - `Admin` role - has full access to the account except that they can't manage roles for Owner users or delete the account.

Reviewer: ```suggestion
- `Admin` role - has full access to the account except that administrators can't delete or update the role of the Owner user and delete the organization account.
```

> - `User` role - has limited access to the account, allowing users to view peers they own and others they can connect with. Users can create personal access tokens for programmatic access.

Reviewer: ```suggestion
- `User` role - has limited access to the account, allowing users to view peers they own and others they can connect to. Users can create personal access tokens for programmatic access.
```
